processors/zipkin/src/main/java/org/hawkular/apm/processor/zipkin/CommunicationDetailsDeriver.java

```diff
@@ -159,4 +164,21 @@ public CommunicationDetails processOneToOne(String tenantId, Span item) throws R
         return ret;
     }
 
+    private void setServiceNames(CommunicationDetails communicationDetails, Span span, ProducerInfo producerInfo) {
+
+        // service name from server span
+        if (span.service() != null) {
+            communicationDetails.getProperties().add(new Property(Constants.PROP_SERVICE_NAME, span.service()));
+        }
+
+        // service name from producer (client span)
+        List<Property> collect = producerInfo.getProperties().stream()
```

Comment:
This part is unnecessary, as the 'service' property in the ProducerInfo will be copied over with line 127/130 above.